gen/generate.toml

```diff
@@ -0,0 +1,23 @@
+[general]
+library_name = "libClp"
+output_file_path = "../lib/libClp.jl"
+# module_name = "libclp"
```

Comment:
Why are some of these commented? Some of the options are pretty opaque. For example how do we know to use `smart_de_anonymize = true`?

Reply:
You can check [Clang's own generator.toml](https://github.com/JuliaInterop/Clang.jl/blob/master/gen/generator.toml) for reference. They are commented because I tested them but end up disabling them.